common/rules.go

```diff
@@ -11,22 +10,6 @@ import (
 	"github.com/emirpasic/gods/sets/treeset"
 )
 
-// Return the default target name for the given language.GenerateArgs.
-// The default target name of a BUILD is the directory name. WHen within the repository
-// root which may be outside of version control the default target name is the repository name.
-func ToDefaultTargetName(args language.GenerateArgs, defaultRootName string) string {
```

Comment:
The idea of defaulting to the dir name is a bad idea so I want to avoid this being promoted in "common" and moving it to the one place it's used in the js language.